.editorconfig

```diff
@@ -0,0 +1,7 @@
+root = true
+
+[*]
+end_of_line = lf
+indent_style = tab
```

Comment:
A bit unrelated maybe, but I added a .editorconfig that helps enforce eslint config - especially tab indentation.